modules/tinymce/src/core/main/ts/keyboard/DeleteBackspaceKeys.ts

```
@@ -111,6 +113,7 @@ const setup = (editor: Editor, caret: Cell<Text | null>): void => {
   editor.on('keydown', (evt: EditorEvent<KeyboardEvent>) => {
     isBackspaceKeydown = evt.keyCode === VK.BACKSPACE;
 
+    debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)